W-17596437-bridgeservicedecomissioning-se (#451)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[May-20-2025]]
== May 20, 2025

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * You can no longer publish an invocable run configuration directly to Salesforce. You can use https://help.salesforce.com/s/articleView?id=platform.flow_build_use_flows_with_mulesoft_rpa.htm&type=5[Anypoint Exchange] or  https://help.salesforce.com/s/articleView?id=platform.api_catalog.htm&type=5[API Catalog] instead.

Reviewer: ```suggestion
* You can no longer publish an invocable run configuration directly to Salesforce. Use https://help.salesforce.com/s/articleView?id=platform.flow_build_use_flows_with_mulesoft_rpa.htm&type=5[Anypoint Exchange] or https://help.salesforce.com/s/articleView?id=platform.api_catalog.htm&type=5[API Catalog] instead.
```

+

> Invocable run configurations already published to Salesforce remain there. The *Published to Salesforce* badges are removed from all published invocable run configurations.

Reviewer: ```suggestion
Invocable run configurations already published to Salesforce remain active. The *Published to Salesforce* badges are removed from all published invocable run configurations.
```
"remain active/available" or "continue working" maybe?

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID
| You can now re-assign bots of deployed invocable run configurations. | W-17807695

> | Invocable confguration runs no longer return an error when they exceed a certain limit of assigned tasks. | W-18380196

Reviewer: ```suggestion
| Process executions no longer return an error when they exceed the number of assigned sessions. | W-18380196
```
Is this more clear? The issue mentions invocable and manual runs.
